Misc/NEWS.d/next/Tests/2023-08-20-11-55-06.gh-issue-107895.kM3neC.rst

```diff
@@ -0,0 +1 @@
+remove dependency of current debug mode
```

Comment:
This doesn't explain what this PR actually does. It also needs proper capitalization and punctuation.

Reply:
I just edited the sentence.